src/main/java/org/gridsuite/modification/server/NetworkModificationController.java

```diff
@@ -72,6 +72,13 @@ public NetworkModificationController(NetworkModificationService networkModificat
         return ResponseEntity.ok().body(networkModificationService.getModifications(groupUuid, true));
     }
 
+    @DeleteMapping(value = "/groups/{groupUuid}/modifications/{modificationUuid}")
+    @Operation(summary = "Get modifications list of a group")
+    @ApiResponse(responseCode = "200", description = "Modification deleted")
+    public ResponseEntity<Mono<Void>> deleteModifications(@Parameter(description = "Group UUID") @PathVariable("groupUuid") UUID groupUuid, @PathVariable("modificationUuid") UUID modificationUuid) {
```

Comment:
should we add this for the second parameter ?:
`@Parameter(description = "Modification UUID")`

Reply:
done